assets/pages/new-contributions-landing/checkoutFormIsSubmittableActions.js

```diff
@@ -89,7 +89,7 @@ const formIsValidParameters = (state: State) => ({
   firstName: state.page.form.formData.firstName,
   lastName: state.page.form.formData.lastName,
   email: state.page.form.formData.email,
-  showOneOffNameFields: state.common.abParticipations.showOneOffNameFields === 'control' || state.page.form.contributionType !== 'ONE_OFF',
+  showOneOffNameFields: state.page.form.contributionType !== 'ONE_OFF',
```

Comment:
Should this variable be renamed to showNameFields? It seems odd to have a variable name showOneOffNameFields which is only true for non one off contributions!

Reply:
excellent point, I'll fix that

Reply:
I'll change to "hasNameFields" because this is actually validating them, not showing them

Reply:
Do you even need this actually? Seeing as you already pass in contributionType. You could change the line on line 74 to say `contributionType === 'ONE_OFF' ?`

Reply:
thanks guys yeah I misread originally but I have renamed and moved it now, have a look 👍